sdk/storage/storage-blob-changefeed/src/models/BlobChangeFeedEvent.ts

```diff
@@ -0,0 +1,32 @@
+export type BlobChangeFeedEventType = "BlobCreate" | "BlobDeleted";
+
+export interface BlobChangeFeedEvent {
+  topic: string;
+  subject: string;
+  eventType: BlobChangeFeedEventType;
+  eventTime: string;
+  id: string; // GUID
+  data: BlobChangeFeedEventData;
+  dataVersion?: string;
+  metadataVersion: string;
+}
+
+
+export type BlobType = "BlockBlob" | "AppendBlob" | "PageBlob";
+
+export interface BlobChangeFeedEventData {
```

Comment:
what about storage diagnostics field? [https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-change-feed?tabs=azure-portal](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-change-feed?tabs=azure-portal)

Reply:
It shouldn't be used by user. So we simply don't expose it..